domains/pearlcz.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "0pearlcz0",
+           "email": "",
+           "discord": "809488731353776149",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.UKtO0TGQ2vw45um_Y1n0Pj66OTCqJlpmyrIFY2TU8nzaMXYf40P4I0jdTeI0begjgJSK98WBExHDHTKQEAljN1LZHpDF0nZQD0xg7uRnqemy3lGceshf0QIs-Tyq5ipavx9CjosXoEMwtiqhmFDKS57sLtI37wtT0f-zambKmBiRFR18gzxV3AuBgpLSASoR-PWbJfMf64BCekXdUrrZ352BxRp9tjrAkxxK7iMcSNJn69mKkMdLlLu92X67WGU5MTKctUfrFOZS_pE1UZyAGzZodmCiS3QiKJMo-ZXvZyy7VzSB4EYgYcLYX1jjsgTmoPl-EXnyk1KAYcVDvEJziQ.oI8DhkP6hv1aeLuXl95Q5w.stm_-qvuO3CsAT1ZZwfjgJdrXx6fbwc8M6ORnJQE6MB6dAWTYyl0bYvBUJUs5nrC4sUl2wg5xoV68td2d4z4CJYYf-sUV_cXyf2gRmCGV94.TGY8u9yodBgKz4fGvldIfQ"
+        },
+    
+        "record": {
+            "CNAME": "ns1.byet.org"
```

Comment:
is this a NameServer (NS)?

in case you can put a NS as CNAME and we don't support NS records